scripts/process-audit-results.js

```
@@ -0,0 +1,52 @@
+// process-audit-results.js
+const fs = require('fs');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)